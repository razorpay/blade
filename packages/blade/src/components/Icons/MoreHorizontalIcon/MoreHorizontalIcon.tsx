import { Svg, Path } from '../_Svg';
import type { IconComponent } from '..';
import useIconProps from '../useIconProps';

const MoreHorizontalIcon: IconComponent = ({ size, color, ...styledProps }) => {
  const { height, width, iconColor } = useIconProps({ size, color });

  return (
    <Svg {...styledProps} width={width} height={height} viewBox="0 0 24 24" fill="none">
      <Path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M4 9C2.34315 9 1 10.3431 1 12C1 13.6569 2.34315 15 4 15C5.65685 15 7 13.6569 7 12C7 10.3431 5.65685 9 4 9ZM9 12C9 10.3431 10.3431 9 12 9C13.6569 9 15 10.3431 15 12C15 13.6569 13.6569 15 12 15C10.3431 15 9 13.6569 9 12ZM17 12C17 10.3431 18.3431 9 20 9C21.6569 9 23 10.3431 23 12C23 13.6569 21.6569 15 20 15C18.3431 15 17 13.6569 17 12Z"
        fill={iconColor}
      />
    </Svg>
  );
};

export default MoreHorizontalIcon;
