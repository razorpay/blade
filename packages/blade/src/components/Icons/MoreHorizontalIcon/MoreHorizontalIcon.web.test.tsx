import MoreHorizontalIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<MoreHorizontalIcon />', () => {
  it('should render MoreHorizontalIcon', () => {
    const { container } = renderWithTheme(
      <MoreHorizontalIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
