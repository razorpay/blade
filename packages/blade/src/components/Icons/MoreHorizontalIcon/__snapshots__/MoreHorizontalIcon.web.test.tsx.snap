// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoreHorizontalIcon /> should render MoreHorizontalIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      clip-rule="evenodd"
      d="M4 9C2.34315 9 1 10.3431 1 12C1 13.6569 2.34315 15 4 15C5.65685 15 7 13.6569 7 12C7 10.3431 5.65685 9 4 9ZM9 12C9 10.3431 10.3431 9 12 9C13.6569 9 15 10.3431 15 12C15 13.6569 13.6569 15 12 15C10.3431 15 9 13.6569 9 12ZM17 12C17 10.3431 18.3431 9 20 9C21.6569 9 23 10.3431 23 12C23 13.6569 21.6569 15 20 15C18.3431 15 17 13.6569 17 12Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
